src/GLib/GLib.jl

```diff
@@ -62,21 +54,15 @@ importall .CompatGLib
 using .CompatGLib.WORD_SIZE
 
 # local function, handles Symbol and makes UTF8-strings easier
-typealias AbstractStringLike Union{AbstractString, Symbol}
+@compat const  AbstractStringLike = Union{AbstractString, Symbol}
 bytestring(s) = String(s)
 bytestring(s::Symbol) = s
-if VERSION >= v"0.5.0-dev+4612"
-    bytestring(s::Ptr{UInt8}, own::Bool) = unsafe_wrap(String, s, ccall(:strlen, Csize_t, (Ptr{UInt8},), s), own)
-else
-    bytestring(s::Ptr{UInt8}, own::Bool) = String(pointer_to_array(s, Int(ccall(:strlen, Csize_t, (Ptr{UInt8},), s)), own))
-end
-if VERSION >= v"0.5.0-dev+4612"
-    bytestring(s::Ptr{UInt8}) = unsafe_string(s)
-elseif VERSION >= v"0.5.0-dev+4200"
-    bytestring(s::Ptr{UInt8}) = String(s)
-else
-    bytestring(s::Ptr{UInt8}) = Base.bytestring(s)
+function bytestring(s::Ptr{UInt8}, own::Bool)
+    # unsafe_wrap(String, s, ccall(:strlen, Csize_t, (Ptr{UInt8},), s), own)
+    @assert own==false #println("own=$own")
+    return bytestring(s)
```

Comment:
3-arguments unsafe_wrap gives deprecation in favor of unsafe_string, which has no 3 arguments method. Is it ok to just let handle this to unsafe_string(s)?